android-components/components/browser/engine-gecko/build.gradle

```
@@ -56,6 +57,7 @@ dependencies {
     implementation project(':concept-fetch')
     implementation project(':support-ktx')
     implementation project(':support-utils')
+    implementation project(':service-nimbus')
```

What about this here:

```suggestion
    implementation(project(':service-nimbus')) {
        exclude group: 'org.mozilla.telemetry', module: 'glean-native'
    }
```

The issue is (I think) that engine-gecko depends on geckoview, which itself provides the glean-native part.
But then nimbus also depends on `glean-native` transitively, but doesn't know about geckoview, so it's pulled in again.
By excluding that explicitly it should fall back to the geckoview-provided one when it's all compiled together.

I ran `./gradlew lint` on this change locally and it seemed to work, but I didn't run any tests.
We do something similar here: https://github.com/mozilla-mobile/firefox-android/blob/8dc9f859a4f75050c2d5f6b020a54963211aadcf/android-components/components/service/sync-logins/build.gradle#L35-L37